tools/release.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+'use strict';
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const { buildDockerImages, pushDockerImage } = require('./docker-utils');
+
+async function main() {
+  const argv = yargs(hideBin(process.argv))
+    .option('defaultNodeDockerVersion', {
+      alias: 'default-node-docker-version',
+      describe: 'Node Docker version for latest tag',
+      demandOption: true,
+    })
+    .option('nodeDockerVersions', {
+      alias: 'node-docker-versions',
+      describe:
+        'Comma-separated list of Node Docker tag versions (e.g., "12-alpine,14-alpine")',
+      demandOption: true,
+      coerce: (opt) => {
+        return opt.split(',');
+      },
+    })
+    .check(({ nodeDockerVersions, defaultNodeDockerVersion }) => {
+      if (nodeDockerVersions.includes(defaultNodeDockerVersion)) {
+        return true;
+      }
+
+      throw new Error(
+        `defaultNodeDockerVersion "${defaultNodeDockerVersion}" was not in list of versions "${nodeDockerVersions}"`,
+      );
```

Comment:
I started writing code to trigger a build if the `defaultNodeDockerVersion` wasn't present in the `nodeDockerVersions`, but it turned into a big mess so I decided to fail early instead

Reply:
Failing early seems logical :)